packages/tools/pluggable-widgets-tools/scripts/mxbuild.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mono:5.20.1
+FROM mono:latest
```

Comment:
Updating this fixed an issue in CI during `apt-get update` where an upstream dependency could not be fetched over https.